lib/ansible/modules/source_control/gitlab_deploy_key.py

```diff
@@ -1,40 +1,41 @@
 #!/usr/bin/python
-# -*- coding: utf-8 -*-
-
-# (c) 2018, Marcus Watkins <marwatk@marcuswatkins.net>
-# Based on code:
-# (c) 2013, Phillip Gentry <phillip@cx.com>
+# (c) 2015, Werner Dijkerman (ikben@werner-dijkerman.nl)
```

Comment:
I'm not sure about copy rights updates, but I think it would be good to describe why these updates are needed?

Reply:
wrong copy/past